client/src/components/AuthForm/SignUpForm/SignUpForm.js

```diff
@@ -39,15 +39,18 @@ const SignUpForm = ({
     return result.length < 4 || signUpForm.password.length < 6;
```

Comment:
why do you need to check if result.length is < 4? 

And why are you returning a boolean from handleChange ??